research/src/pages/documentation/get-involved.mdx

```diff
@@ -10,6 +10,16 @@ We welcome all types of contributions from attending a meeting, to giving feedba
 
 ## Getting Started
 
+The best place to start is by participating in one or more of our communication channels:
+
+[Discord server](https://discord.gg/DEWjhSw) - Where we hold general discussions between our meetings
+[Open UI Github](https://github.com/WICG/open-ui) - Where we house the website's repo and open issues
```

Comment:
Let's adjust this a bit:

* Discord: Ask a question, learn about what's happening, be alerted of upcoming meetings and opportunities to get involved in the Open UI community.
* Open UI Github: Our repo where we track issues, progress on spec proposals, and publish our research for public consumption
* Open UI Telecon Agenda: We have weekly telecons at 11:00 AM Pacific Standard Time, you can find previous minutes and upcoming agendas here. If you want to get added to the calendar invite contact [Greg Whitworth](https://twitter.com/gregwhitworth) or [Una Kravets](https://twitter.com/Una) on [Discord](https://discord.gg/DEWjhSw) or Twitter
* Open UI Community Group: In order to add substantial contributions to Open UI you need to join the Open UI CG

Reply:
Updated!